### PR TITLE
Make relationshipProperties error messages more helpful

### PR DESCRIPTION
**Pull request by mjfwebb**

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Thanks for the Neo4j GraphQL Toolbox updates.

The Neo4j GraphQL Toolbox has now been torn down - reopening this PR will republish it.

File: packages/graphql/src/schema/validation/custom-rules/directives/relationship.ts [hunk 1]
```diff
@@ -90,29 +90,42 @@ export function verifyRelationshipArgumentValue(
...
-            const relationshipPropertiesInterface = extra.objects.filter(
+            const relationshipPropertiesInterface = extra.interfaces.filter(
+                (i) =>
+                    i.name.value.toLowerCase() === propertiesValue.toLowerCase() &&
+                    i.kind === Kind.INTERFACE_TYPE_DEFINITION
+            );
+
+            if (relationshipPropertiesInterface.length > 0) {
+                throw new DocumentValidationError(
+                    `@relationship.properties invalid. @relationship.properties must be applied to an interface. Note, since version 5.0.0, the @relationshipProperties directive must be applied to a type and not an interface.`,
```
Comment (darrellwarde) [inline packages/graphql/src/schema/validation/custom-rules/directives/relationship.ts]: Middle sentence is a bit weird! Maybe:

```suggestion
                    `@relationship.properties invalid. The @relationshipProperties directive must be applied to a type and not an interface, a breaking change introduced in version 5.0.0.`,
```

???

**Review by darrellwarde — CHANGES_REQUESTED** [review]
Looks generally good, just a comment around one of the error messages!

**Review by darrellwarde — APPROVED** [review]
LGTM!

**Pull request merged**